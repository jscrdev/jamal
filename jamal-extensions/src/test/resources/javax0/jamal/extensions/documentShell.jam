{@JShell
    void hello(){
        System.out.println("Hello, " + world);
    }
}{@script hello/JShell(world)=hello();}
{hello My Dear}

{@JShell
    void hallo(){
        System.out.println("Hallo, " + you);
    }
}
{#block
{@script helloBlock/JShell(a,b)=hallo(
);hello()}
{@export helloBlock}}
{hello/My World/You}