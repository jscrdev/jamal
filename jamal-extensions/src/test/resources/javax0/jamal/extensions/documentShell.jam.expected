
Hello, My Dear




Hello, My World/You
